### PR TITLE
Fix coverage tests for Kernel PR 1203

### PR DESCRIPTION
**Pull request by aggarg**
Description
-----------
Fix coverage tests for Kernel PR https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1203.

Test Steps
-----------
Run coverage tests.

Checklist:
----------
- [x] I have tested my changes. No regression in existing tests.
- [x] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1203

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jasonpcarroll** [comment]
Please fix failing unit tests.

File: FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c [hunk 1]
```diff
@@ -1095,7 +1095,7 @@ void test_coverage_prvCreateIdleTasks_name_within_max_len( void )
 
     /* Test Verifications */
     xIdleTask = ( TCB_t * ) xIdleTaskHandles[ 0 ];
-    TEST_ASSERT_EQUAL_STRING( configIDLE_TASK_NAME, xIdleTask->pcTaskName );
+    TEST_ASSERT_EQUAL_STRING( "IDLE longX0", xIdleTask->pcTaskName );
```
Comment (AniruddhaKanhere) [inline FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c:1098]:
```suggestion
    TEST_ASSERT_EQUAL_STRING( "IDLE longXX", xIdleTask->pcTaskName );
```

File: FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c [hunk 2]
```diff
@@ -1158,7 +1158,7 @@ void test_coverage_prvCreateIdleTasks_name_too_long( void )
     xIdleTask = ( TCB_t * ) xIdleTaskHandles[ 0 ];
 
     /* Test Verifications */
-    TEST_ASSERT_EQUAL_STRING_LEN( configIDLE_TASK_NAME,
+    TEST_ASSERT_EQUAL_STRING_LEN( "IDLE long 0",
```
Comment (AniruddhaKanhere) [inline FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c:1161]:
```suggestion
    TEST_ASSERT_EQUAL_STRING_LEN( "IDLE long n",
```

**Review by AniruddhaKanhere — CHANGES_REQUESTED** [review]
@aggarg the suggested changes should fix the tests.

**Review by AniruddhaKanhere — DISMISSED** [review]

**Comment by AniruddhaKanhere** [comment]
@aggarg, do we also need to update the submodule of the Kernel?

**Review by jasonpcarroll — APPROVED** [review]

**Review by jasonpcarroll — CHANGES_REQUESTED** [review]
Update submodule pointer for the FreeRTOS-Kernel

File: FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c (hunk 2 above)
Comment (aggarg) [reply FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c]: The tests will be fixed after this PR is merged - https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1203.

File: FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c (hunk 1 above)
Comment (aggarg) [reply FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice_mock/covg_multiple_priorities_no_timeslice_mock_utest.c]: The tests will be fixed after this PR is merged - https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1203.

**Review by AniruddhaKanhere — APPROVED** [review]

**Review by jasonpcarroll — APPROVED** [review]
Approved with the notion that this needs to merge before updating the submodule pointer

**Pull request merged**